src/utils/tokens.js

```diff
@@ -118,6 +118,33 @@ const tokens = {
   getNFTFee() {
     return 1;
   },
+
+  /**
+   * Returns the token signatures on storage
+   * This is only used for hw wallets.
+   *
+   * @memberof Tokens
+   * @inner
+   */
+  getTokenSignatures() {
+    const tokenSignatures = hathorLib.storage.getItem('wallet:token:signatures');
+    if (!tokenSignatures) return {};
+    // if (tokenSignatures === null) return {};
+    return tokenSignatures;
+  },
+
+  /**
+   * Add a token signature to storage, overwriting if exists
+   * This is only used for hw wallets.
+   *
```

Comment:
Missing params and return types in the docstring

Reply:
added